ion/src/device/shared/drivers/reset.cpp

```diff
@@ -49,7 +49,7 @@ void jump(uint32_t jumpIsrVectorAddress) {
   // Disable cache before reset
   Ion::Device::Cache::disable();
 
-  /* Shutdown all clocks and peripherals to mimic a hardware reset. */
+  /* Shutdown all clocks and periherals to mimic a hardware reset. */
```

Comment:
?

Reply:
? Changed correct spelling to wrong spelling